robottelo/test.py

```diff
@@ -238,11 +240,21 @@ def take_screenshot(self, testmethodname):
         testmethodname = "_".join(testmethodname)
         # Creates dir_structure depending upon the testmethodname.
         directory = testmethodname
-        if not os.path.exists("/tmp/screenshots/" + directory):
-            os.makedirs("/tmp/screenshots/" + directory)
-        self.browser.save_screenshot('/tmp/screenshots/{dir}/'
-                                     'screenshot-{date}.png'
-                                     .format(dir=directory, date=filename))
+        os_ver = self.server_name.split(".")[0].split("-")[2]
+        if not os.path.exists('{screenshot_dir}/{date}/{ver}/{dir}/'
+                              .format(screenshot_dir=self.screenshot_dir,
+                                      date=date, ver=os_ver, dir=directory)):
+            os.makedirs('{screenshot_dir}/{date}/{ver}/{dir}/'
```

Comment:
This format:

```python
'{screenshot_dir}/{date}/{ver}/{dir}/'.format(screenshot_dir=self.screenshot_dir, date=date,ver=os_ver, dir=directory)
```

Could be saved to a variable for later usage as it is being used more than once